### ISSUE TITLE
Vue Filemanager should allow setting thumbnail for MVW

### ISSUE THREAD

**Issue by escowles** [pulibrary/figgy#783]

**Comment by sdellis** [comment]
I understand the need, but I'm not sure how we want to go about this.

This was a conversation in Slack a couple weeks ago: 

> shaune [4:37 PM]
> Now that I think of it, I wonder if we should include thumbnails in mvw referenced manifest. It seems silly to load an entire manifest just to get to the thumbnail for the filemanager, no?
> 
> tpendragon [4:38 PM]
> :+1: Seems reasonable to ignore for now.
> can always be an improvement down the line
> 
> shaune [4:38 PM]
> Ok, just use a default then?
> 
> tpendragon [4:40 PM]
> Sure

So, unless we load the entire manifest for a thumbnail, the thumbnails won't even show up in the Filemanager.  Is there any way to add just the thumb to the manifests (brief references) in a MVW? Should I open a separate ticket for this?

A few ways to do this, but perhaps the best/most performant is to create a "select thumbnail" button/link that launches a modal and asynchronously load all that child's canvases allowing for thumbnail selection. Any objections to that approach, or did you have something else in mind?

**Comment by sdellis** [comment]
Thinking about this more, perhaps instead of a modal we could "expand" the volume on some click/event and load all the images right into the filemanager then. No sense in creating a modal gallery, when there's a gallery already there. Of course, this leads to questions as to whether users should be able to edit/rearrange the children within the MVW filemanager UI.

**Comment by hackmastera** [comment]
Hm. seems like people might feel worried or confused about whether stuff they had already done was getting saved before all the content in the gallery changed.

**Comment by escowles** [comment]
I think a much more limited feature would do the job here: allowing the user to select which volume should be used to select the thumbnail would cover all the cases I have heard about.  I have not heard any requests like "use this image for the thumbnail of volume 1, but use this different image for the thumbnail of the MVW as a whole".  So a reasonable default (first image of first volume) and allowing selection of which volume's thumbnail to use is all we need here.

**Comment by sdellis** [comment]
Ok, that works @escowles. I guess the only remaining issue is that in order to show the volume thumbnails for selection, I need to dereference and load all volume manifests in the MVW filemanager. Can we stick the thumbnails on the brief reference in the manifest? That would simplify this issue greatly.

**Comment by sdellis** [comment]
Blocked by #952
